lib/lightspeed/item.rb

```diff
@@ -58,5 +58,10 @@ class Item < Lightspeed::Resource
     def prices
       @prices ||= Lightspeed::Prices.new(self.Prices)
     end
+
+    def default_vendor
```

Comment:
rather than this, you can add , `relationships: DefaultVendor: :Vendor` to the end of the relationships array, (see `Parent: :Category` for an example)

so this would be `relationships :ItemMatrix, :Category, :Images, DefaultVendor: :Vendor`